test/user.spec.js

```diff
@@ -0,0 +1,56 @@
+const request = require('supertest');
+const app = require('../app');
+
+const userCreate = {
+  name: 'Joe',
+  last_name: 'Doe',
+  email: 'joedoe@wolox.co',
+  password: 'holamundo2019'
+};
+
+const userParams = {
+  name: '',
+  last_name: 'Montoya',
+  email: 'joedoe@wolox.co',
+  password: 'holamundo'
+};
+
+const userPassword = {
+  name: 'Felipe',
+  last_name: 'Montoya',
+  email: 'joedoe@wolox.co',
+  password: 'holamundo'
+};
+
+describe('User creation', () => {
```

Comment:
In your tests you are only checking status codes. In user creation for example you should test that the created user is in the database

Reply:
Also, the tests' names should be more explicit. For example 'creating user with valid properties and non existing email is valid'